ocaml/testsuite/tests/typing-layouts/parsing_beta.compilers.reference

```diff
@@ -11,7 +11,7 @@ Error: Syntax error: layout expected.
 Line 2, characters 9-15:
 2 | type t = float#;;
              ^^^^^^
-Error: The extension "layouts" is disabled and cannot be used
+Error: The extension "layouts" is disabled at maturity "alpha" and cannot be used
```

Comment:
I think this error is a little confusing (though the old one wasn't great either, so I'm not particularly blaming you).  It would be nice to say something like:
```
"float#" requires the "alpha" version of the "layouts" extension, which is disabled.
```
Maybe we could achieve this by having `Disabled_extension` carry an optional message prefix string so we can provide a little context-relevant information?

Reply:
Your idea is nice. I don't want to see it all the way through right now, but I can at least make the layouts error message here a little more intelligible.